extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/CdcSourceP.java

```diff
@@ -413,10 +414,17 @@ private static void log(ILogger logger, String name, String property, Object val
     }
 
     private static long extractTimestamp(SourceRecord record) {
```

Comment:
record is a keyword in Java 17. So it would be better to rename it to something else. For example sourceRecord